addons/web/static/src/views/graph/graph_model.js

```diff
@@ -444,12 +441,24 @@ export class GraphModel extends Model {
             // There is a currency aggregate
             if (measures.length > 2) {
                 const currencies = group[measures[1]];
```

Comment:
`measures[1]` bof. This code could be written in a more readable and robust way I think

Reply:
`measures.at(-2)` is actually even worse